c10/core/DispatchKeySet.h

```diff
@@ -8,6 +7,28 @@
 
 namespace c10 {
 
+struct FunctionalityOffsetAndMask {
+    // empty constructor shouldn't be used; only needed to initialize
+    // the array before populating it.
+    FunctionalityOffsetAndMask() {}
+    FunctionalityOffsetAndMask(uint16_t offset, uint16_t mask) :
+        offset(offset), mask(mask) {}
```

Comment:
I think you can just omit the constructors entirely, but I've certainly been wrong before :)

Reply:
Yeah, I tried deleting one / both and the compiler wasn't happy. (I don't think the compiler generates the 2-arg constructor, and adding it explicitly deletes the 0 arg constructor).